drivers/usb/usbport/device.c

```diff
@@ -1443,15 +1560,18 @@ USBPORT_RemoveDevice(IN PDEVICE_OBJECT FdoDevice,
                      IN ULONG Flags)
 {
     PUSBPORT_DEVICE_EXTENSION FdoExtension;
+    PUSB2_TT_EXTENSION TtExtension;
+    ULONG ix;
+    KIRQL OldIrql;
 
     DPRINT("USBPORT_RemoveDevice: DeviceHandle - %p, Flags - %x\n",
            DeviceHandle,
            Flags);
 
     FdoExtension = FdoDevice->DeviceExtension;
 
-    if ((Flags & USBD_KEEP_DEVICE_DATA) ||
-        (Flags & USBD_MARK_DEVICE_BUSY))
+    if ((Flags & USBD_KEEP_DEVICE_DATA) != 0 ||
+        (Flags & USBD_MARK_DEVICE_BUSY) != 0)
```

Comment:
Original code (without the != 0) is actually OK, this is precisely what it does.

Reply:
Yeah, this is just more explicit. I don't know why Vadim wanted to change it by I don't disagree so I left it in.